node-sdk-sample/README.md

```diff
@@ -0,0 +1,54 @@
+# Zowe Node SDK Sample
+
+## Requirements
+
+You can clone the repository using `git`:
+
+```
+git clone https://github.com/zowe/zowe-client-samples.git
+```
+
+You can install the requirements in a virtual environment with following commands:
+
+```shell
+cd node-sdk-sample
+npm install
+npm run build
+```
+
+## Quickstart
+
+After you install the package, import the class SampleSDK from modules.
+
+Below is a simple script of how to run the sample SDK, assuming the script file is located under the `node-sdk-sample/src` folder:
+
+```typescript
+import { Session } from "@zowe/imperative";
+import { SampleSdk } from "./main";
+
+(async () => {
+    const session: Session = new Session({
+        hostname: "jsonplaceholder.typicode.com",
+    });
+    const sdk = new SampleSdk(session);
+    console.log("\n=== Retrieving a Single Post ===");
+    console.log(await sdk.getPost(1));
+})();
+
+```
+
+## Running The Sample
+
+~/zowe-client-sample/node-sdk-sample should be the working directory
```

Comment:
Thoughts on putting the working dir in inline code blocks (single backticks) and adjusting the wording to match Zowe Docs / other SDKs?

```suggestion
With `~/zowe-client-sample/node-sdk-sample` as the working directory, issue the following command to run the sample:
```